.github/workflows/build.yml

```diff
@@ -42,11 +42,8 @@ jobs:
       - name: Run xcode-select
         run: sudo xcode-select --switch /Applications/Xcode_15.0.1.app
       - name: Install dependencies
-        # Remove pip call after Chromium 119
         run: |
-          brew install coreutils ninja
-          pip3 install -U setuptools
-          pip3 install -U html5lib
+          brew install coreutils ninja python3 --overwrite
```

Comment:
This can be one-lined again, as in
```diff
-        run: |
-          brew install coreutils ninja python3 --overwrite
+        run: brew install coreutils ninja python3 --overwrite
```
And it should probably be added to every step, not just the first one. setuptools only needed to be added to the first step because the only script that used it was executed once at the beginning of the build, but there are other python scripts that might be affected by differences between 3.11 and 3.12 that could run in later steps.